crates/universal-sierra-compiler-api/src/lib.rs

```diff
@@ -59,16 +59,26 @@ pub fn compile_sierra(sierra_contract_class: &Value, sierra_type: &SierraType) -
 
     compile_sierra_at_path(
         &Utf8PathBuf::from(temp_sierra_file.path().to_string_lossy().to_string()),
+        false,
         sierra_type,
     )
 }
 
 pub fn compile_sierra_at_path(
     sierra_file_path: &Utf8PathBuf,
+    is_saved_file: bool,
     sierra_type: &SierraType,
 ) -> Result<String> {
-    let mut usc_command = UniversalSierraCompilerCommand::new();
+    // Skip printing details when compiling unsaved sierra
+    // which occurs during test execution for some cheatcodes e.g. `replace_bytecode`
+    if is_saved_file {
+        println!(
+            "Compiling Sierra to Casm ({})",
+            sierra_file_path.canonicalize_utf8()?
+        );
+    }
```

Comment:
I think we can slightly improve styling displayed message 😄 .

Before:
![image](https://github.com/user-attachments/assets/3bd98deb-5949-47b9-b23f-54da6a4c7731)

After:
![image](https://github.com/user-attachments/assets/f8252a92-1624-462f-8589-b61c3c6a86bc)

Note that I've also added `println!("\n")` in [this line](https://github.com/foundry-rs/starknet-foundry/blob/bdec9564cf5d0c05d83a66acb2945eb8d426cdb2/crates/forge/src/run_tests/workspace.rs#L70).

Here's an example code that achieves this
```rust
if is_saved_file {
        println!(
            "{} ({})",
            style("Compiling Sierra to Casm").bold(),
            sierra_file_path.canonicalize_utf8()?,
    );
}
```

Reply:
nit: Just for the sake of uniformity, can we use `CASM` instead of `Casm` 😅 ? In the vast majority of places in the docs we use screaming case version.

Reply:
Renamed both to `SIERRA` and `CASM`